ace_network.py

```diff
@@ -239,6 +234,18 @@ def create_from_split_state_dict(cls, encoder_state_dict, head_state_dict):
         encoder_state_dict: encoder state dictionary
         head_state_dict: scene-specific head state dictionary
         """
+
+        # https://github.com/nianticlabs/ace/issues/20
+        # To make the model scriptable, converting to nn.moduleList has renamed some model parameters
+        # The following code backports the old pretrained weight format to the new format if it is necessary
+        cur_head_block = 0
+        while(f'{cur_head_block}c0.weight' in encoder_state_dict):
+            for weight_type in ["weight", "bias"]:
+                for weight_num in range(3):
+                    encoder_state_dict[f'res_blocks.{cur_head_block}.{weight_num}.{weight_type}'] = encoder_state_dict[f'{cur_head_block}c{weight_type}.{weight_type}']
+                    del encoder_state_dict[f'{cur_head_block}c{weight_type}.{weight_type}']
```

Comment:
I don't think this works as intended: `encoder_state_dict[f'{cur_head_block}c{weight_type}.{weight_type}']`

Shouldn't it be: `encoder_state_dict[f'{cur_head_block}c{weight_num}.{weight_type}']` ?